meta-mender-core/classes/mender-artifactimg.bbclass

```diff
@@ -18,6 +18,31 @@ ARTIFACTIMG_FSTYPE_DEFAULT = "ext4"
 ARTIFACTIMG_NAME ??= "${ARTIFACTIMG_NAME_DEFAULT}"
 ARTIFACTIMG_NAME_DEFAULT = "${IMAGE_LINK_NAME}"
 
+MENDER_ARTIFACT_NAME_DEPENDS ?= ""
+
+MENDER_ARTIFACT_PROVIDES ?= ""
+MENDER_ARTIFACT_PROVIDES_GROUP ?= ""
+
+MENDER_ARTIFACT_DEPENDS ?= ""
+MENDER_ARTIFACT_DEPENDS_GROUPS ?= ""
+
+apply_arguments () {
```

Comment:
Happy to receive some suggestions for improvements on this one

Reply:
I like having a generic function. :-)